functions/page-index/page-index.js

```diff
@@ -0,0 +1,20 @@
+module.exports = [
+  {
+    title: "Get started with PostgreSQL functions",
```

Comment:
I have a few concerns with this title. Would appreciate input from @davidkohn88 at least.

- "Functions" are a programatic term, of which Postgres supports them, but not all of the things we cover are functions. Our first example, `ORDER BY`, is not a function in the classical sense and I wouldn't expect anyone to refer to it as such. How else could we say "Postgres basics" without saying "basics"?

- I don't think we should have two "get started with" sections one after the other. The main thrust, IMO, should be getting people started with TimescaleDB. I'm not sure that a logical/great progression should be 
    * Install
    * get started with TimescaleDB
    * get started with Postgres

Reply:
Would "commands" work better than "functions"? I was under the impression that "function" was more accurate here. I struggled a lot to come up with something (anything!) other than "Getting Started" but couldn't think of anything that really worked. Any suggestions welcome!

Reply:
"Working with PostgreSQL" maybe? I'm also drawing a blank

Reply:
I could be convinced on that ... @ryanbooz ?

Reply:
Yeah, functions is highly misleading, I think you could say getting started with Postgres functionality?  Commands is also fine, but maybe doesn't convey what you want? Statements is another way of saying that (ie, I can run a `SELECT` statement or an `INSERT` statement). Query is another useful term, it's usually applied to `SELECT`s. Sometimes they're called SQL keywords (ie `FROM` or `ORDER BY` or `AS` or `IN`), some of the keywords have associated clauses (ie `FROM` clause or `ORDER BY` clause) in which case it refers to the part of the query. 

Honestly, I think something that explains this naming might be an incredibly useful concept doc and might really help with writing the rest of this stuff.

Reply:
Thanks for that, David! I'm actually leaning towards "keywords", based on this. Could "Working with PostgreSQL keywords" work?

Reply:
Hmmm...that still sounds like very odd phrasing to me. Mostly because you don't use the keywords, they are a class of thing that appears in a query, or a statement or a command, but I wouldn't say you could use them on their own? I don't know if that makes sense. You could say something like PostgreSQL keywords and how to use them? Or understanding PostgreSQL keywords? But in this case I'd say your phrasing just strikes me as odd, it's not like it's actually misleading like functions. 

If you want to go with your phrasing, I'd say it's less odd to say something like "Working with PostgreSQL commands" rather than keywords. 

The keywords one sort of feels like saying, working with your nose to understand the world instead of saying smell things? I don't know if that analogy was remotely useful.

Reply:
That is helpful, thanks David. I like "Working with PostgreSQL commands".

Reply:
Updated.